cli/Makefile

```diff
@@ -1,44 +1,19 @@
 TURBO_VERSION = $(shell cat ../version.txt | sed -n '1 p')
 TURBO_TAG = $(shell cat ../version.txt | sed -n '2 p')
 
-EXT :=
-ifeq ($(OS),Windows_NT)
-	UNAME := Windows
-	EXT = .exe
-else
-	UNAME := $(shell uname -s)
-endif
-
 CLI_DIR = $(shell pwd)
 
-# We do not set go-turbo as a dependency because the Rust build.rs
-# script will call it for us and copy over the binary
-turbo:
-	cargo build -p turbo
-
-turbo-prod:
-	cargo build --release --manifest-path ../crates/turborepo/Cargo.toml
-
-turbo-capnp:
-	cd ../crates/turborepo-lib/src/hash && capnp compile -I std -ogo proto.capnp && mv ./proto.capnp.go ../../../../cli/internal/fs/hash/capnp
-
-install: | ./package.json
-	pnpm install --filter=cli
-
-corepack:
-	which corepack || npm install -g corepack@latest
-	corepack enable
-
-build: install
-	cd $(CLI_DIR)/../ && pnpm build:turbo
-	cd $(CLI_DIR)/../ && pnpm install --filter=create-turbo && pnpm turbo-prebuilt build --filter=create-turbo...
-	cd $(CLI_DIR)/../ && pnpm install --filter=@turbo/codemod && pnpm turbo-prebuilt build --filter=@turbo/codemod...
-	cd $(CLI_DIR)/../ && pnpm install --filter=turbo-ignore && pnpm turbo-prebuilt build --filter=turbo-ignore...
-	cd $(CLI_DIR)/../ && pnpm install --filter=@turbo/workspaces && pnpm turbo-prebuilt build --filter=@turbo/workspaces...
-	cd $(CLI_DIR)/../ && pnpm install --filter=@turbo/gen && pnpm turbo-prebuilt build --filter=@turbo/gen...
-	cd $(CLI_DIR)/../ && pnpm install --filter=eslint-plugin-turbo && pnpm turbo-prebuilt build --filter=eslint-plugin-turbo...
-	cd $(CLI_DIR)/../ && pnpm install --filter=eslint-config-turbo && pnpm turbo-prebuilt build --filter=eslint-config-turbo...
-	cd $(CLI_DIR)/../ && pnpm install --filter=@turbo/types && pnpm turbo-prebuilt build --filter=@turbo/types...
```

Comment:
Filtered installs were unnecessary as we already did a top level install everywhere this is run.

Combined all of these runs into a single run with multiple filters. Was also able to get rid of `...` as this results in the same tasks getting run.